### PR TITLE
Adding new languages to list

### PR DESCRIPTION
**Pull request by McLoo**
### Contributor checklist

- [x] I am following the [Code Style Guidelines](https://github.com/WhisperSystems/Signal-Android/wiki/Code-Style-Guidelines)
- [x] I have tested my contribution on these devices:
 * Virtual device (Nexus 5), Android 6.0.1
- [X] My contribution is fully baked and ready to be merged as is
- [X] I ensure that all the open issues my contribution fixes are mentioned in the commit message of my first commit using the `Fixes #1234` [syntax](https://help.github.com/articles/closing-issues-via-commit-messages/)
- [X] I have made the choice whether I want the [BitHub reward](https://github.com/WhisperSystems/Signal-Android/wiki/BitHub-Rewards) or not by omitting or adding the word `FREEBIE` in the commit message of my first commit

----------

### Description
Adding the following languages, which have at least 90% of their translalation finished
- [Estonian](https://en.wikipedia.org/wiki/Persian_language) 100% 
- [Albanian](https://en.wikipedia.org/wiki/Albanian_language) 97% _(A bit ironic, to put Shqip next to Serbian...)_
- [Norwegian Bokmål](https://en.wikipedia.org/wiki/Bokm%C3%A5l) 97% 
- [Norwegian Nynorsk](https://en.wikipedia.org/wiki/Nynorsk) 97% 
- [Basque](https://en.wikipedia.org/wiki/Basque_language) 94% 
- [Ganda](https://en.wikipedia.org/wiki/Luganda) 90% 
- [Persian](https://en.wikipedia.org/wiki/Persian_language)  90%

@moxie0  maybe we should also remove languages which dropped below 50%?!

### CODE REVIEW

**Comment by RiseT** [comment]
I wonder why there is a translation for "Norwegian (no)" in addition to Norwegian Bokmål and Norwegian Nynorsk.
As far as I understand it, these are the only 2 languages officially used in Norway.

**Comment by McLoo** [comment]
All three are (slightly) different...
Could be, that NO is an older translation and NB and NN are new translated.
NO is at 92% maybe we should kick that one out. A native norwegian speaker would be of help ^^


NO
  ```
  <string name="attachment_type_selector__location">Lokasjon</string>

  </plurals>
  <string name="expiration_hours_abbreviated">%dt</string>
  <plurals name="expiration_days">
    <item quantity="one">1 dag</item>
    <item quantity="other">%d dager</item>
  </plurals>
  <string name="expiration_days_abbreviated">%dd</string>
  <plurals name="expiration_weeks">
    <item quantity="one">1 uke</item>
    <item quantity="other">%d uker</item>
  </plurals>


  <string name="device_link_fragment__link_device">Koble enhet</string>
```

NN
```
  <string name="attachment_type_selector__location">Plassering</string>

  <string name="expiration_hours_abbreviated">%dt</string>
  <plurals name="expiration_days">
    <item quantity="one">1 dag</item>
    <item quantity="other">%d dagar</item>
  </plurals>
  <string name="expiration_days_abbreviated">%dd</string>
  <plurals name="expiration_weeks">
    <item quantity="one">1 veke</item>
    <item quantity="other">%d veker</item>
  </plurals>

  <string name="device_link_fragment__link_device">Kopla til eining</string>
```

NB
```
  <string name="attachment_type_selector__location">Plassering</string>

  <string name="expiration_hours_abbreviated">%dt</string>
  <plurals name="expiration_days">
    <item quantity="one">1 dag</item>
    <item quantity="other">%d dager</item>
  </plurals>
  <string name="expiration_days_abbreviated">%dd</string>
  <plurals name="expiration_weeks">
    <item quantity="one">1 uke</item>
    <item quantity="other">%d uker</item>
  </plurals>


  <string name="device_link_fragment__link_device">Koble til enhet</string>
```

**Comment by RiseT** [comment]
Just checked at Transifex. For what it's worth, Nynorsk does only have one single translator currently.

**Comment by RiseT** [comment]
I'm asking one of the Norwegian translators.

**Comment by RiseT** [comment]
@McLoo OK. I've contacted a Norwegian translator who has been kind enough to reply.

Essentially he says that "NO" should get removed, as even for a Norwegian native it's unclear what just "Norwegian" is supposed to mean.

He's under the impression that new translators start translating the "NO" strings, while actually they should haven chosen either Bokmål or Nynorsk. Bokmål is the language version spoken by the majority, though.

**Comment by unhammer** [comment]
>  For what it's worth, Nynorsk does only have one single translator currently.

OTOH, I now have Transifex-notifications on for Signal-Android, and use the app regularly :-)

**Comment by unhammer** [comment]
Is there anything blocking this?

**Comment by RiseT** [comment]
The PR currently doesn't remove NO ("Norsk"). @McLoo ...?

**Comment by RiseT** [comment]
@unhammer Since your a native-speaker:
Do the language names "Norsk bokmål" and "Norsk nynorsk" make sense in Norwegian, or would it be better to call them "Norsk (bokmål)" and "Norsk (nynorsk)"...? Just wondering.

File: res/values/arrays.xml [hunk 1]
```diff
@@ -21,16 +23,21 @@
       <item>Japanese 日本語</item>
       <item>Kannada ಕನ್ನಡ</item>
       <item>Korean 한국어</item>
+      <item>Luganda</item>
       <item>Magyar</item>
       <item>Macedonian македонски јазик</item>
       <item>Nederlands</item>
       <item>Norsk</item>
```
Comment (RiseT) [inline res/values/arrays.xml]: Remove

File: res/values/arrays.xml [hunk 2]
```diff
@@ -59,16 +68,21 @@
       <item>ja</item>
       <item>kn_IN</item>
       <item>ko</item>
+      <item>lg</item>
       <item>hu</item>
       <item>mk</item>
       <item>nl</item>
       <item>no</item>
```
Comment (RiseT) [inline res/values/arrays.xml]: Remove

File: res/values/arrays.xml (hunk 1 above)
Comment (RiseT) [inline res/values/arrays.xml]: Remove

**Comment by unhammer** [comment]
I wouldn't write "Norsk nynorsk" in a sentence, while "Norsk (nynorsk)" works fine. However, lists and headlines allow for slightly different grammar from the rest of language, so I don't think it's that important if you'd like to avoid parentheses.

(In Norwegian, we just say "Nynorsk" or "Bokmål", though in a language list people would typically be looking for something starting with "Nor…".)

File: res/values/arrays.xml (hunk 1 above)
Comment (RiseT) [inline res/values/arrays.xml]: Remove

File: res/values/arrays.xml [hunk 3]
```diff
@@ -21,16 +23,21 @@
       <item>Japanese 日本語</item>
       <item>Kannada ಕನ್ನಡ</item>
       <item>Korean 한국어</item>
+      <item>Luganda</item>
       <item>Magyar</item>
       <item>Macedonian македонски јазик</item>
       <item>Nederlands</item>
       <item>Norsk</item>
+      <item>Norsk bokmål</item>
```
Comment (RiseT) [inline res/values/arrays.xml]: Change to "Norsk (bokmål)"

File: res/values/arrays.xml [hunk 4]
```diff
@@ -21,16 +23,21 @@
...
+      <item>Norsk nynorsk</item>
```
Comment (RiseT) [inline res/values/arrays.xml]: Change to "Norsk (nynorsk)"

File: res/values/arrays.xml (hunk 2 above)
Comment (RiseT) [inline res/values/arrays.xml]: Remove

**Comment by McLoo** [comment]
Finally back to business...
"fixed" Norsk

**Comment by RiseT** [comment]
Great. One question remains, though. Who's responsible for removing "Norwegian" aka "Norsk" aka "NO" from Transifex? It would be an unfortunate waste of resources if some translators were still translating these now redundant strings (vs. choosing between bokmål and nynorsk):

https://www.transifex.com/moxie/signal-android/language/no/

**Comment by RiseT** [comment]
@unhammer I see you're the only member of Nynorsk at Transifex. I guess it wouldn't hurt if you asked someone (e. g. one of the team managers...?) in order to get promoted to "reviewer" or even better "coordinator" for Nynorsk.

**Comment by unhammer** [comment]
I asked https://www.transifex.com/user/profile/leibi/ :)
And thanks for adding nn!

**Pull request closed without merging**